HISTORY.md

```diff
@@ -1,6 +1,16 @@
 # Release History - `price-oracle`
 
 
+## 0.9.1 (2023-02-24)
+
+- Makes set up step explicit in the demo instructions e45cfc9b05f40ec0b708a1e7bc594196edfeef7b
+- Bumps `open-aea` and `open-autonomy` frameworks #48 && #52
```

Comment:
Please, specify versions

Reply:
Done in 7ff8a8c926397c1606b87c67ae713dc5a12cbfea.